retro/reco.py

```diff
@@ -419,23 +427,23 @@ def _reco_event(self, event, method, save_llhp, filter, save_estimate=True):
             filter = filter.strip()
             print("filter: '{}'".format(filter))
 
-        print("Running {} reconstructioni on event".format(method))
+        print("Running {} reconstruction on event".format(method))
 
         if filter and not eval(filter):  # pylint: disable=eval-used
             print(
                 "filter evaluates to False; skipping event (index {})".format(
                     event.meta["event_idx"]
                 )
             )
-            return -1
+            return FitStatus.InsufficientHits.value
```

Comment:
Why are you returning `.value` when you can simply return the `IntEnum`? (which behaves as an `IntEnum` for things that know how to deal with it, and unless I'm mistaken, behaves as an `int` for things that only know how to work with integers)

Reply:
I first tried to return `FitStatus.InsufficientHits` and it did not like it, so I had to change it to `.value`